src/__test__/out_object.spec.js

```diff
@@ -28,46 +28,187 @@ describe('Command', () => {
         expect(outObj).toBeInstanceOf(OutObject);
     });
 
-    it('#property() returns a value set by phantom', async () => {
-        let page = await phantom.createPage();
-        let outObj = phantom.createOutObject();
+    describe('#on', () => {
+        'use strict';
+        describe('("onResourceRequested", true)', () => {
+            it('returns a value set by phantom', async () => {
+                let page = await phantom.createPage();
+                let outObj = phantom.createOutObject();
 
-        await page.property('onResourceReceived', function(response, out) {
-            out.lastResponse = response;
-        }, outObj);
+                await page.property('onResourceRequested', true, function(requestData, networkRequest, out) {
```

Comment:
@pboling why you passing `true` to `property`? That's supposed to be for `on` only according to the read me.